cache/cache_test.go

```diff
@@ -436,58 +436,116 @@ func TestCacheCompactionResync(t *testing.T) {
 		{Key: []byte("foo"), Value: []byte("old_value"), ModRevision: 5, CreateRevision: 5, Version: 1},
 	})
 
+	mw.responses <- clientv3.WatchResponse{
```

Comment:
I don't understand any of the changes below. They lack any explanation nor they make any coherent story.